charts/hono/Chart.yaml

```diff
@@ -17,7 +17,7 @@ description: |
   interacting with them in a uniform way regardless of the device communication protocol.
 version: 1.8.0
 # Version of Hono being deployed by the chart
-appVersion: 1.8.0
+appVersion: 1.8.1
```

Comment:
I guess you wanted to bump the *version* property instead ...

Reply:
yes, fixed now.